mne_bids/tests/test_read.py

```diff
@@ -317,6 +317,32 @@ def test_handle_coords_reading():
     with pytest.raises(RuntimeError, match='Channels do not correspond'):
         raw_test = read_raw_bids(bids_fname, bids_root)
 
+    # make sure montage is set if there are coordinates w/ 'n/a'
+    raw.info['bads'] = []
+    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
+    electrodes_dict = _from_tsv(electrodes_fname)
+    for axis in ['x', 'y', 'z']:
+        electrodes_dict[axis][0] = 'n/a'
+        electrodes_dict[axis][3] = 'n/a'
+    _to_tsv(electrodes_dict, electrodes_fname)
+
+    # set channel status as 'good' explicitly
+    channels_fname = _find_matching_sidecar(bids_fname, bids_root,
```

Comment:
why don't you just do `read_raw_bids` here and check in `raw.info['chs']` ? The less you use private functions in tests, the better.

Reply:
See the comment down below.